reference/features/search_parameters.md

```diff
@@ -238,6 +238,10 @@ Configures which attributes will be retrieved in the returned documents.
 
 If no value is specified, `attributesToRetrieve` uses the [`displayedAttributes` list](/reference/features/settings.md#displayed-attributes), which by default contains all attributes found in the documents.
 
+::: note
+If a field is not added to `displayedAttributes`, it will not be retrieved on search, even if it is present in the `attributesToRetrieve` parameter.
```

Comment:
There's something funny going on here that makes the whole sentence very complex for me. I think it's tied to us giving the context on the first clause ("If a field is not added etc") and then adding more context on the final clause ("even if it is present etc").

Is it just me? Is this simpler?

```suggestion
If an attribute has been removed from `displayedAttributes`, `attributesToRetrieve` will silently ignore it and the field will not appear in your returned documents.
```

Reply:
I like this better too!

Reply:
Huh. I've been staring at these two sentences for more than five minutes. I think I prefer Maryam's original—it appeals to my robot brain—but I also see the potential for confusion that Gui pointed out. There's a real double negative problem.

Here's my minimalistic attempt, FWIW:

```
Attributes must be present in `displayedAttributes` to be retrievable with `attributesToRetrieve`.
```